syftr/__init__.py

```diff
@@ -1,3 +1,6 @@
-from importlib.metadata import version as _version
+try:
+    from importlib.metadata import version as _version
 
-__version__ = _version("syftr")
+    __version__ = _version("syftr")
+except Exception:
+    __version__ = "0.0.0"
```

Comment:
If that corresponds to when using from a repo, there's already a way to get the version of `syftr` package: https://github.com/datarobot/syftr/blob/main/syftr/ray/runtime_env.py#L21

Reply:
the metadata functionality is apparently used for the syftr package but that's not working on the cluster. What would you want to change?

Reply:
perhaps it is available in [ray.runtime_env.RuntimeEnv](https://docs.ray.io/en/latest/ray-core/api/doc/ray.runtime_env.RuntimeEnv.html)
cc @alex-dr

Reply:
![image](https://github.com/user-attachments/assets/f258a14f-3f0e-4053-b179-2c365a9b796f)

@alex-dr @nvdv